File: Projet_final/projetFinalSpringBoot/src/main/java/com/exemple/model/Gestionnaire.java
package com.exemple.model;

import java.util.Set;

import javax.persistence.DiscriminatorValue;
import javax.persistence.Entity;
import javax.persistence.OneToMany;

@Entity
@DiscriminatorValue("Gestionnaire")
public class Gestionnaire extends User {
	@OneToMany(mappedBy = "gestionnaire")
	private Set<Cursus> cursus;

	public Set<Cursus> getCursus() {
		return cursus;
	}

	public void setCursus(Set<Cursus> cursus) {
		this.cursus = cursus;
	}
}

File: Projet_final/angularBT/src/app/model/gestionnaire.ts
import { Cursus } from './cursus';
import { User } from './user';
export class Gestionnaire extends User{
  constructor(private _cursus: Cursus = null) {
    super();

  }
  
  public get cursus(): Cursus {
    return this._cursus;
  }
  
  public set cursus(_cursus: Cursus) {
    this._cursus =_cursus;
  }

}